corehq/toggles.py

```diff
@@ -1835,3 +1835,10 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_CUSTOM,
     [NAMESPACE_USER]
 )
+
+DISABLE_CASE_UPDATE_RULE_SCHEDULED_TASK = StaticToggle(
+    'disable_case_update_rule_task',
+    'Disable the `run_case_update_rules` periodic task',
```

Comment:
Would it make sense to add a bit more explanation so future reviewers of this code have more context on why it was added (to answer questions like "can it be removed?")?
```suggestion
    'Disable the `run_case_update_rules` periodic task '
    'while investigating database performance issues.',
```